sql/core/src/main/scala/org/apache/spark/sql/scripting/SqlScriptingExecutionNode.scala

```diff
@@ -80,6 +89,11 @@ class SingleStatementExec(
    */
   var isExecuted = false
 
+  /**
+   * Data returned after execution.
+   */
+  var data: Option[Array[Row]] = None
```

Comment:
let's use something other than `data` - it seems pretty generic. something like `result` seems more suitable?